### ISSUE TITLE
Sample Trainer and Decoder for this

### ISSUE THREAD

**Issue by abaheti95** [AuCson/PyTorch-Batch-Attention-Seq2seq#9]
Hi,
Can you also provide trainer and decoder files for the batch model if possible. If you have tested this code then you must have them. Kindly upload those files if possible.

**Comment by pravn** [comment]
Yes could you put up data and main file, please? It would be quite useful.
